modules/internal/ChapelDistribution.chpl

```diff
@@ -760,12 +763,24 @@ module ChapelDistribution {
     var buf: [bufDom] idxType;
     var cur = 0;
 
+    var dataSorted = false;
+    var isUnique = false;
+
     proc init(size, param rank: int, obj) {
       this.rank = rank;
       this.obj = obj;
       bufDom = {0..#size};
     }
 
+    proc init(size, param rank: int, obj, dataSorted, isUnique) {
```

Comment:
Similar to the comment above, these two flags could be added to the initializer above with default values being `false`, I think.